packages/opentelemetry/src/init.ts

```diff
@@ -125,7 +125,7 @@ instrumentations.forEach((i) => {
 
 let tracerProvider: NodeTracerProvider | null;
 
-export interface OpenTelemetryConfig {
+export interface OpenTelemetryConfigEnabled {
   openTelemetryEnabled: boolean;
```

Comment:
If you want to correctly discriminate between these two types:

```suggestion
  openTelemetryEnabled: true;
```

Reply:
I did this originally, but wasn't sure if it would be a source of issues in the other call to the init function in server.js once that's type-checked. I changed it to true for now as you suggested.

Reply:
Ah, that's actually a fair point - I didn't think about that. I think that will in fact be an issue, but that's a bridge we can cross when we come to it.